dot/services_integration_test.go

```diff
@@ -39,6 +45,55 @@ func TestCreateStateService(t *testing.T) {
 	require.NotNil(t, stateSrvc)
 }
 
+func newStateServiceWOMock(t *testing.T) *state.Service {
+	t.Helper()
+
+	stateConfig := state.Config{
+		Path:      t.TempDir(),
+		LogLevel:  log.Info,
```

Comment:
nit let's silent logs for tests by default

```suggestion
		LogLevel:  log.Error,
```

Reply:
Updated.